msg/hover_thrust_estimate.msg

```diff
@@ -1,4 +1,5 @@
-uint64 timestamp		# time since system start (microseconds)
+uint64 timestamp                # time since system start (microseconds)
+uint64 timestamp_sample
```

Comment:
A small comment describing `timestamp_sample` would be nice.
Is the `timestamp_sample` actually used or useful? If not feel free to leave it away.

Reply:
It's the timestamp of the accel data (& estimate). I'll try to write something that doesn't sound like a trivial tautology.